receipt_chroma/receipt_chroma/embedding/formatting/word_format.py

```
@@ -79,81 +81,139 @@ def format_word_context_embedding_input(
+        # All edges - return all edges
+        return ["<EDGE>"] * context_size, ["<EDGE>"] * context_size
+
+    # The middle word is likely the target (assuming context_size words on each side)
+    # But we don't know which one, so we'll extract context_size from each side
+    # For backward compatibility, return first word from each side
+    left_tokens = []
+    right_tokens = []
+    found_word = False
+
+    for token in tokens:
+        if token == "<EDGE>":
```

Ruff (0.14.6)

Possible hardcoded password assigned to: "token"

(S105)